utils/utils.go

```diff
@@ -8,16 +8,12 @@ import (
 	"fmt"
 	"math/big"
 	"net/url"
+	"strings"
 
 	"github.com/ethereum/go-ethereum/common"
 )
 
 const (
-	// Default stake threshold for aggregate signature verification. (67%)
-	// TODO: This should be made configuration on the VM level.
-	DefaultQuorumNumerator   = 67
-	DefaultQuorumDenominator = 100
-
```

Comment:
Do we need to complete https://github.com/ava-labs/awm-relayer/issues/8 for release?

Reply:
Good call. I'll update that issue now.